specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-01-01-preview/Metadata.json

```diff
@@ -628,6 +636,63 @@
       "type": "string",
       "format": "date"
     },
+    "metadataCustomVersion": {
+      "description": "The custom version of the content. A optional free text",
+      "type": "string"
+    },
+    "metadataContentSchemaVersion": {
+      "description": "Schema version of the content. Can be used to distinguish between different flow based on the schema version",
+      "type": "string"
+    },
+    "metadataIcon": {
+      "description": "the icon identifier. this id can later be fetched from the solution template",
+      "type": "string"
+    },
+    "metadataThreatAnalysisTactics": {
```

Comment:
>metadataThreatAnalysisTactics [](http://example.com/codeflow?start=5&length=29)

Is this is a well known list? Or do you expect this to be dynamic?

Reply:
We expect this to be pretty dynamic. It changes in a high frequency.